cloud/providers/ec2/ec2spot.go

```diff
@@ -138,7 +138,26 @@ func (cloudManager *EC2SpotManager) OnUp(host *host.Host) error {
 	ec2handle, client := getUSEast(*cloudManager.awsCredentials)
 	defer util.PutHttpClient(client)
 
-	return attachTags(ec2handle, tags, spotReq.InstanceId)
+	resources := []string{
+		spotReq.InstanceId,
+	}
+
+	instance, err := getInstanceInfo(ec2handle, spotReq.InstanceId)
+	if err != nil {
+		err = errors.Wrap(err, "problem getting underlying request")
+		grip.Error(err)
```

Comment:
Does this need to log? I think the calling code would log it already

Reply:
this is currently the pattern in this package, and while I hope you're right, I'm not sure that that's consistently true.